telicent_lib/adapter.py

```diff
@@ -207,7 +207,9 @@ def run(self) -> None:
                             default_headers = [
                                 ('Exec-Path', self.generated_id),
                                 ('Request-Id', request_id),
-                                ('traceparent', carrier.get('traceparent', ''))
+                                ('traceparent', carrier.get('traceparent', '')),
+                                ('Data-Source-Name', str(self.source_name)),
+                                ('Data-Source-Type', str(self.source_type))
                             ]
```

Comment:
Andy had suggested adding a data source id as well, we don't currently capture this from a user so you may want to look at how we could do that

Reply:
This could be something that is obtained from the Telicent Catalog in due course.  Whether that is supplied to an adapter (as the Data-Source-Name and Data-Source-Type are, for example) or an adapter retrieves it from the Telicent Catalog I don't know at this point.
For now I'm happy for there to be a placeholder property which is not populated (i.e. shows nothing) until we get the mechanics of *how* to populate it correctly up and running.
Happy to discuss further as required.